startup.sh

```diff
@@ -1,4 +1,14 @@
 #!/bin/bash
 
 bundle install
-bundle exec rails s -p 3005
+
+if [[ $1 == "--live" ]] ; then
+  GOVUK_APP_DOMAIN=www.gov.uk \
+  GOVUK_WEBSITE_ROOT=https://www.gov.uk \
+  PLEK_SERVICE_CONTENT_STORE_URI=https://www.gov.uk/api \
```

Comment:
still haven't figured out what that should actually be?
i presume
search.publishing.service.gov.uk
or
rummager.publishing.service.gov.uk

but nether of them seem to exist